addons/spectator/RscDisplayEGSpectator.hpp

```diff
@@ -0,0 +1,5 @@
+class RscDisplayEGSpectator
+{
+    scriptName = "rscdisplayegspectator";
```

Comment:
If this really needs to stay lower case for compatibility lets comment that.
```suggestion
    scriptName = "rscdisplayegspectator"; // keep lowercase for compatibility with BI stuff
```

Reply:
Turns out it works fine with proper casing, so I've changed it.